install-conda-dependencies.sh

```diff
@@ -1,16 +1,20 @@
 #!/bin/bash
 
 #
-# This script will install conda packages (in the current conda env) to satisfy 
+# This script will install conda packages (in the current conda env) to satisfy
 # mirgecom dependency requirements. It optionally accepts an argument that is
 # the path to a file which should have a list of packages to install.
 #
 # Usage: install-conda-dependencies.sh [path/to/optional/packages.txt]
 #
- 
+
 set -o errexit
+
+#shellcheck disable=SC1090
+source "$MY_CONDA_DIR"/bin/activate "$MY_CONDA_ENV"
```

Comment:
This contradicts the comment above: It doesn't use the "current" env, it activates a new one. IMO it should not activate anything.

Reply:
"Fixed" in 717cc47. The problem here is that for some reason, install-conda-dependencies.sh does not pick up the environment activated in install.sh; the only workaround I found is to explicitly export PATH in install.sh.

Reply:
> install-conda-dependencies.sh does not pick up the environment activated in install.sh

Huh. Super weird.